nav2_controller/src/controller_server.cpp

```diff
@@ -268,8 +273,8 @@ ControllerServer::on_activate(const rclcpp_lifecycle::State & /*state*/)
   }
   vel_publisher_->on_activate();
   action_server_->activate();
```

Comment:
Lets try this one last time, this needs to go between the vel publisher and the action server, so that its available by the time the action server is active, but in the same code block as all activations